Assets/Scripts/Boss/CarrotAttack/CarrotManager.cs

```diff
@@ -0,0 +1,70 @@
+using System.Collections;
+using System.Collections.Generic;
+using GLTF.Schema;
+using UnityEngine;
+
+public class CarrotManager : MonoBehaviour
```

Comment:
```suggestion
public class CarrotAttackManager : MonoBehaviour
```
CarrotManager라고만 하면 CarrotControl과 구분이 잘 안되어서,
전체적인 당근 공격을 관장하는 매니저 느낌이 나게, CarrotAttackManager로 네이밍을 바꿔보는게 어떨까 싶어요